### ISSUE TITLE
Problem with installation in Kali linux 4.13.0

### ISSUE THREAD

**Issue by Bluewild93** [astsam/rtl8812au#73]
I updated my Kali linux to 4.13.0. And i can't install the driver. Hot to fix the problem? Can you make a driver that we can install in kali 4.13.0?

**Comment by kimocoder** [comment]
@Bluewild93 use [this](https://github.com/aircrack-ng/rtl8812au/tree/v4.3.21) which got the kernel support

**Comment by Bluewild93** [comment]
Thank you. But
I installed your driver 4.3.21, it works perfect for the first time when i put it in the monitor mode then sent DeAuth to my target devices. My target devices was disconnect from my wifi. But i couldn't get any handshakes. The second time, i couldn't get anything from the airodump-ng 
![img_0088](https://user-images.githubusercontent.com/14236569/31361515-fd048976-ad53-11e7-9c9a-a8018548ba8a.JPG)

**Comment by alsetema** [comment]
Same here, cannot compile on Lububtu 17.10, running 4.13 (tried with the versions 4.3.21 of the driver and version 5 as well. The only one that seems to compile is the one from aircrack-ng, which doesnt seem to support TXpower, which i really need.

**Comment by kimocoder** [comment]
Use the link provided above, it supports newer kernels including kernel v4.13

**Comment by alsetema** [comment]
@kimocoder Correct, that is the one that works for me, yet TXpower is not working unfortunately.
What's good is that using the driver from aircrack-ng, IW command now detects the device

**Comment by kimocoder** [comment]
be sure to use the v4.3.21 branch, the v5.1.5 seems a bit dirty

**Comment by alsetema** [comment]
@kimocoder I am, i do think there is an issue open about it.

**Comment by ajayv707** [comment]
same problem in monitor mode my device not showing
